### PR TITLE
Mark all secrets missing in subcommand

### PR DESCRIPTION
**Pull request by btwonion**
Adds a subcommand for marking every secret in the current dungeons room as missing.

### CODE REVIEW

**Comment by kevinthegreat1** [comment]
I would put this under `markAsMissing`, maybe as `markAsMissing all`.

**Comment by btwonion** [comment]
done

File: src/main/java/de/hysky/skyblocker/skyblock/dungeon/secrets/DungeonManager.java [hunk 1]
```diff
@@ -361,7 +362,15 @@ private static void loadJson(BufferedReader reader, Map<String, JsonElement> map
     }
 
     private static RequiredArgumentBuilder<FabricClientCommandSource, Integer> markSecretsCommand(boolean found) {
-        return argument("secretIndex", IntegerArgumentType.integer()).executes(context -> {
+        return argument("secretIndex", IntegerArgumentType.integer()).suggests((provider, builder) -> {
+            if (isCurrentRoomMatched()) {
+                int secretCount = currentRoom.getSecretCount();
+                for (int i = 1; i < secretCount; i++) {
```
Comment (kevinthegreat1) [inline src/main/java/de/hysky/skyblocker/skyblock/dungeon/secrets/DungeonManager.java:368]: Could be a nice stream but whatever.

**Review by kevinthegreat1 — APPROVED** [review]
Lgtm, thanks for your pr.

File: src/main/java/de/hysky/skyblocker/skyblock/dungeon/secrets/DungeonManager.java (hunk 1 above)
Comment (btwonion) [reply src/main/java/de/hysky/skyblocker/skyblock/dungeon/secrets/DungeonManager.java]: if this is what you meant here you go

Comment (kevinthegreat1) [reply src/main/java/de/hysky/skyblocker/skyblock/dungeon/secrets/DungeonManager.java]: Wait a minute, this should be inclusive. So `IntStream.rangeClosed(1, room.getSecretCount())` using `rangeClosed` and without the `- 1`.

**Review by kevinthegreat1 — APPROVED** [review]

**Comment by kevinthegreat1** [comment]
This is pretty overdue, so we can include this in 1.22.

**Pull request merged**